src/react-components/scene-ui.js

```diff
@@ -273,28 +272,11 @@ class SceneUI extends Component {
               </a>
             </div>
           </div>
-        </div>
-        <div className={styles.info}>
-          <div className={styles.name}>{this.props.sceneName}</div>
-          <div className={styles.attribution}>{attributions}</div>
-        </div>
-        <IfFeature name="enable_spoke">
```

Comment:
Wasn't able to check the design notes so I assume its covered in there, but just in case, just making sure this was intentionally removed.

Reply:
Correct, spoke call out is intentionally removed. Thanks for checking.